tests/python_tests/ov_genai_test_utils.py

```diff
@@ -25,6 +25,7 @@ def get_models_list():
         "microsoft/phi-1_5",
         "microsoft/phi-2",
         "THUDM/chatglm2-6b",
+        "THUDM/chatglm3-6b", # no beam_search
```

Comment:
Does every test pass with `THUDM/chatglm3-6b`? If not, please, mark the failing tests to be skipped.

Reply:
Skips must happen only for that particular model